lib/context-menu-handlers.js

```diff
@@ -126,6 +126,23 @@ function init(panel) {
     label: contextMenuLabel,
     context: [
       cm.URLContext(['*.vimeo.com']),
+      cm.SelectorContext('.iris_video-vital__overlay.iris_link-box')
+    ],
+    contentScript: `self.on('click', function (node, data) {
+      self.postMessage('https://vimeo.com' + node.href);
+    });`,
+    onMessage: (url)=> {
```

Comment:
I haven't figured out a way to specify two contexts for a single item. Though that would be really nice. I tried to do that in a previous issue a couple of weeks ago.

I can pull the `contentScript` out into a variable.